math/fastor/include/algebra/math/impl/fastor_vector.hpp

```diff
@@ -0,0 +1,57 @@
+/** Algebra plugins, part of the ACTS project
+ *
+ * (c) 2020-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/math/common.hpp"
+#include "algebra/qualifiers.hpp"
+
+namespace algebra::fastor::math {
+
+/** Get a normalized version of the input vector
+ *
+ * @param v the input vector
+ **/
+template <typename scalar_t, auto N>
+ALGEBRA_HOST inline Fastor::Tensor<scalar_t, N> normalize(
+    const Fastor::Tensor<scalar_t, N> &v) {
+
+  return Fastor::norm(v);
+}
+
+/** Dot product between two input vectors
+ *
+ *
+ * @param a the first input vector
+ * @param b the second input vector
+ *
+ * @return the scalar dot product value
+ **/
+template <typename scalar_t, auto N>
+ALGEBRA_HOST_DEVICE inline auto dot(
+    const Fastor::Tensor<scalar_t, N> &a,
+    const Fastor::Tensor<scalar_t, N> &b) {
+  return Fastor::inner(a, b);
+}
+
+/** Cross product between two input vectors
+ *
```

Comment:
You can remove one of the empty lines in the comment block here

Reply:
Fixed.